src/test/unit/sanitize.test.js

```diff
@@ -204,4 +205,25 @@ describe('sanitizePII', function() {
       expect(extensions.baseURL.length).toEqual(0);
     }
   });
+
+  it('should sanitize all the preference read values inside network markers', function() {
+    const profile = processProfile(createGeckoProfile());
+    const PIIToRemove = getRemoveProfileInformation({
+      shouldRemovePreferenceValues: true,
+    });
+
+    const sanitizedProfile = sanitizePII(profile, PIIToRemove).profile;
+    for (const thread of sanitizedProfile.threads) {
+      for (let i = 0; i < thread.markers.length; i++) {
+        const currentMarker = thread.markers.data[i];
+        if (
+          currentMarker &&
+          currentMarker.type &&
+          currentMarker.type === 'PreferenceRead'
+        ) {
+          expect(currentMarker.prefValue).toBeFalsy();
+        }
+      }
+    }
+  });
```

Comment:
I'm not so confident with this test: it would pass even if there's no marker `PreferenceRead`, it's not explicit here.
I think I'd use here the same `getProfileWithMarkers` function you used in the other test, so that it's pretty clear which profile we handle here.
(I'm not sure why we used `createGeckoProfile` in this test file to be honest...)

By using a profile you control better, I think you can then do an easier test with:
```js
const thread = profile.threads[0];
expect(thread.markers).toHaveLength(1);
expect(thread.markers.data[0]).toEqual({
  type: 'PreferenceRead',
  // etc...
  value: ''
});
```

Reply:
Yes! Thank you so much for pointing this out. I wrote a test that didn't test anything and that would always pass. Thanks again for your eagle eyes.

The version that I am posting now is, I think, better!

Looking forward to your feedback. Thanks again!